### PR TITLE
Add some news

### LINKED ISSUE #7: News: FlashFest

### LINKED ISSUE #8: News: MCProHosting

### PR DESCRIPTION
**Pull request by mounderfod**
Closes #7, closes #8.

### CODE REVIEW

File: April23/news.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# News
+
+## FlashFest organised for April Fools’ snapshot
+
+This month of course comes with April Fools' Day, and that means a Minecraft April Fools' update! In this year's offering we saw a feature in which players regularly vote on "proposals", each of which would make a small change to the game, ranging from changing the colour of grass to... turning the game French? C'est bizarre. Anyway, with this snapshot came an impromptu ModFest, dubbed a "FlashFest". The competition gave contestants 24 hours to make anything they could targeting this snapshot. Some of the brilliant and wildly surreal submissions included a "Bottle of Gender" (what), a mod that allows proposals to be converted to items and then shot out of a gun (???) and a mod with a proposal which instils a sudden generosity into villagers, making them offer all trades for free (the most unrealistic one by far). The competition was a great success, and we hope to see more of these FlashFests in the future - if you want to see all the submissions, go and check out ModFest's Discord (https://discord.gg/gn543Ee).
```
Comment (Jamalam360) [inline April23/news.md:5]: What if, Oxford comma :flushed: 

`[...] included a "Bottle of Gender" (what), a mod that allows proposals to be converted to items and then shot out of a gun (???), and a mod with a proposal which instils [...]`

Personally am a fan of Oxford commas (i.e. commas before the and in this case), thoughts?

File: April23/news.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+
+## MCProHosting faces potential layoffs
+
+The “world leader” in server hosting appears to be in trouble, according to a now deleted tweet. The tweet, which namechecks fellow hosting company Nitrado, states that “all” the staff got laid off, although this may be hyperbolic, given that there are still staff members in the service’s Discord. The mention to Nitrado is significant as their parent company acquired MCProHosting in 2021, which could mean that MCProHosting could be closing, or is downsizing significantly. If the tweet is true, it is a worrying development, and also a reminder that server hosting companies can often be unreliable. Make your backups, people!
```
Comment (Jamalam360) [inline April23/news.md:9]: `The mention to Nitrado is significant as their parent company acquired MCProHosting in 2021, which could mean that MCProHosting could be closing, or is downsizing significantly.`

This sentence is a bit confusing, had to read it 2 or 3 times. Maybe:

`The mention to Nitrado is significant as their parent company acquired MCProHosting in 2021, which could mean that MCProHosting could be closing or downsizing significantly`.

I feel like that makes it flow better personally.

**Review by Jamalam360 — COMMENTED** [review]
Sounds good, just 2 suggestions.

---

Also, maybe we bring in a formatter to limit markdown to 120 characters per line? Most editors wrap it automatically, but it makes reviewing on GitHub weird because you leave a comment on the entire paragraph rather than the specific line with the issue. 

Any files that go to the website repo will get formatted by prettier anyways, so it might make sense to format here as well.

File: April23/news.md (hunk 1 above)
Comment (mounderfod) [reply April23/news.md:5]: Yeah I agree with you, missed the comma here.
I'll probably write up a style guide soonish

File: April23/news.md (hunk 2 above)
Comment (mounderfod) [reply April23/news.md:9]: Will do, also replacing `mention to` with `mention of` for grammatical correctness

**Review by Jamalam360 — APPROVED** [review]

File: April23/news.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+April Fools' update! In this year's offering we saw a feature in which players
+regularly vote on "proposals", each of which would make a small change to the
+game, ranging from changing the colour of grass to... turning the game French?
+C'est bizarre. Anyway, with this snapshot came an impromptu ModFest, dubbed a
+"FlashFest". The competition gave contestants 24 hours to make anything they
+could targeting this snapshot. Some of the brilliant and wildly surreal
+submissions included a "Bottle of Gender" (what), a mod that allows proposals to
+be converted to items, and then shot out of a gun (???) and a mod with a
+proposal which instils a sudden generosity into villagers, making them offer all
+trades for free (the most unrealistic one by far). The competition was a great
```
Comment (sschr15) [inline April23/news.md:15]: > and a mod with a proposal which instils a sudden generosity into villagers, making them offer all trades for free (the most unrealistic one by far)

hey look it's me
(also typo "instills")

Comment (mounderfod) [reply April23/news.md:15]: :tiny_potato:
I'll fix it now

Comment (mounderfod) [reply April23/news.md:15]: wait actually @sschr15 it's instills in American English and instils in British English
so now i'm not sure which to use

Comment (sschr15) [reply April23/news.md:15]: style guide when it comes

Comment (mounderfod) [reply April23/news.md:15]: time for another PR :p

Comment (mounderfod) [reply April23/news.md:15]: The new style guide (#11) now recommends British English.

**Pull request merged**